universe/auto_syncer.go

```diff
@@ -531,6 +518,134 @@ func (f *FederationEnvoy) SetAllowPublicAccess() error {
 	)
 }
 
+// tryFetchServers attempts to fetch the set of universe servers in the
+// federation.
+func (f *FederationEnvoy) tryFetchServers() ([]ServerAddr, error) {
+	ctx, cancel := f.WithCtxQuit()
+
+	fedServers, err := f.cfg.FederationDB.UniverseServers(
+		ctx,
+	)
+	if err != nil {
+		cancel()
+
+		err := fmt.Errorf("unable to fetch set of universe servers: %v",
+			err)
+		f.reportErr(err)
```

Comment:
We should remove this (depends on order of which PRs land first), as is it'll make the entire daemon shutdown if we can't read the set of servers for w/e reason. So we should just log a `Warnf` here.

Reply:
K will mirror what you have up.